probe-rs/src/debug/snapshots/probe_rs__debug__debug_info__test__unwinding_in_exception_trampoline.snap

```diff
@@ -36,8 +36,3 @@ Frame:
  source_location:
 None
  frame_base:      None
-Frame:
- function:        <unknown function @ 0x0000013c>
```

Comment:
What happened to those? :)

Reply:
Those were "incorrect" duplicate frames in the older code. It is correct that only one of them remains :)